pkg/steps/new_e2e.go

```diff
@@ -265,3 +271,73 @@ func (s newE2ETestStep) runPods(ctx context.Context, pods []coreapi.Pod, notifie
 	}
 	return nil
 }
+
+type openshiftInstallerTestStep struct {
```

Comment:
I don't understand what is going on in this commit

Reply:
First of all, in the best scenario it is intended to be removed. Up to this point, the implementation is completely generic and has no E2E- (or even Openshift-) specific bits.

The code in `openshiftInstallerTestStep` is everything I still don't know how to make generic that is required for the E2E AWS test.. It is also a possible implementation where different test types would share the step back-end but have test-specific extensions, but I would prefer if we could make all that part of configuration / step implementation.